manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: firealpaca.com.FireAlpaca
+PackageVersion: 2.8.7.0
+PackageLocale: en-US
+Publisher: firealpaca.com
+PackageName: FireAlpaca
+License: freeware
+ShortDescription: FireAlpaca Setup

```

Comment:
Please provide a better description of what FireAlpaca does

Reply:
I've added a better description as requested.